sdk/confidentialledger/azure-confidentialledger/CHANGELOG.md

```diff
@@ -1,12 +1,23 @@
 # Release History
 
-## 1.0.0b2 (Unreleased)
+## 1.0.0 (Unreleased)
+
+GA Data Plane Python SDK for Confidential Ledger.
 
 ### Bugs Fixed
+- Fixes [[confidentialledger] mypy: function has duplicate type signatures (#23356)](https://github.com/Azure/azure-sdk-for-python/issues/23356)
```

Comment:
this mypy comment doesn't render correctly, and I don't think it's actually needed, we don't advertise typing / linting fixes